website/settings/defaults.py

```diff
@@ -1958,3 +1959,32 @@ class CeleryConfig:
 DS_METRICS_BASE_FOLDER = None
 REG_METRICS_OSF_TOKEN = None
 REG_METRICS_BASE_FOLDER = None
+
+
+@enum.unique
+class StorageLimits(enum.IntEnum):
+    OVER_CUSTOM = -1
+    DEFAULT = 0
+    APPROACHING_PRIVATE = 4.5 * 10 ** 9
```

Comment:
Two things: 

1. These might be better to work from as if they were GB instead of Bytes, because when we set the custom storage limits in the admin app, it will be much more user friendly for the admins to just say 6 rather than 6000000000.
2. I've gotten approval for the `APPROACHING` limits to just be considered 90% of the `OVER` limits, rather than setting an explicit `APPROACHING` limit. The reason for this is so that we can calculate a reasonably `APPROACHING` limit even when we have custom `OVER` limits.